pkg/cri/resource-manager/control/rdt/doc.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rdt
+
+var configHelp = `
+Resource Manager RDT enforcement controller.
+
+The RDT controller enforces container policy RDT decisions. The controller
+takes the assigned RDT class of a container and puts all processes in the
+containers' cgroup to that class using the resctrl pseudo-filesystem. If
```

Comment:
nit: **container's** instead of **containers'** (?)

Reply:
Fixed.